doc/source/whatsnew/v0.24.0.txt

```diff
@@ -345,6 +421,7 @@ Interval
 ^^^^^^^^
 
 - Bug in the :class:`IntervalIndex` constructor where the ``closed`` parameter did not always override the inferred ``closed`` (:issue:`19370`)
+- Bug in the ``IntervalIndex`` repr missing a trailing comma at the end of the "data" section (:issue`20611`)
```

Comment:
what does this mean? what is the 'data' section? that's an internal reference

Reply:
Cleaned this up.  IIRC this was a side issue that @TomAugspurger found when he originally made the PR.  A comma was missing after the list of intervals in the repr.